doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst

```diff
@@ -47,7 +49,7 @@ the ``backoff`` is selected, |Percona| implementation will be used.
 Multi-threaded LRU flusher
 ==========================
 
-|Percona Server| :rn:`5.7.10-3` has introduced a true multi-threaded LRU
+|Percona Server| :rn:`5.7.10-3` introduced a true multi-threaded LRU
```

Comment:
No need for specific version in previous series

Reply:
The comment has not been addressed

Reply:
Reference to 5.7 removed